src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,37 @@
+.stopwatch {
+  position: relative;
+  border: 20px solid $main-color;
+  border-radius: 50%;
+  width: 600px;
+  height: 600px;
+  box-shadow: inset 0 0 15px black;
+  background-color: $center-stopwatch-color;
+
+  &:before {
+    content: "";
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
```

Comment:
you can use mixin for creating a circle since you've used it twice

```
@mixin circle($width, $color) {
  width: $width;
  height: $width;
  background: $color;
  border-radius: $width/2;
}

.circle {
  @include circle(30px, $center-color);
}
```